Core/Core/Discussions/DiscussionDetailsViewController.swift

```diff
@@ -157,9 +158,18 @@ public class DiscussionDetailsViewController: ScreenViewTrackableViewController,
         webView.handle("like") { [weak self] message in self?.handleLike(message) }
         webView.handle("moreOptions") { [weak self] message in self?.handleMoreOptions(message) }
         webView.handle("ready") { [weak self] _ in self?.ready() }
+        let rootURL = URL.Paths.Offline.courseSectionResourceFolderURL(
+            sessionId: env.currentSession?.uniqueID ?? "",
+            courseId: course.first?.id ?? "",
+            sectionName: isAnnouncement ? OfflineFolderPrefix.announcements.rawValue : OfflineFolderPrefix.discussions.rawValue,
+            resourceId: topicID
+        )
+        webView.loadFileURL(URL.Directories.documents, allowingReadAccessTo: URL.Directories.documents)
         webView.loadHTMLString(
             "<style>\(DiscussionHTML.css)</style>",
-            baseURL: env.api.baseURL.appendingPathComponent("\(context.pathComponent)/discussion_topics/\(topicID)")
+            baseURL: offlineModeInteractor?.isNetworkOffline() == true ?
```

Comment:
`isNetworkOffline` only returns the current network status, we'd need to check `isOfflineModeEnabled` which also checks for the feature flag.